src/test/resources/pages/studentFeedbackSubmitPageFullyFilled.html

```diff
@@ -3933,26 +3933,26 @@ <h4 class="modal-title">
             <div class="col-sm-12">
               <table>
                 <tbody>
-                <tr>
-                  <td>
-                    <div class="radio">
-                      <label class="bold-label">
-                        <input name="responsetext-26-0" type="radio" value="Team 2">
-                        Team 2
-                      </label>
-                    </div>
-                  </td>
-                </tr>
-                <tr>
-                  <td>
-                    <div class="radio">
-                      <label class="bold-label">
-                        <input name="responsetext-26-0" type="radio" value="Team 3">
-                        Team 3
-                      </label>
-                    </div>
-                  </td>
-                </tr>
+                  <tr>
```

Comment:
Extraneous change, needs to be reverted.

Reply:
I was reviewing another PR #8742  and got the same problem where this exact change in indentation was made because tests were run in GODMODE. 

I am assuming Godmode indents the HTML page as well. And I guess this page wasn't indented properly initially causing these changes to be made in around 4 HTML pages whenever Godmode changes anything else in these test resource web pages.

@darrenwee Do you think an issue needs to be made to correct these webpages indentation?

Reply:
I experienced it too in #8687. Can open and issue and do a quick fix?

Reply:
Oo, thanks for bringing this up! I didn't realize this was happening elsewhere too.

@nidhi98gupta In the mean time, you can resolve the merge conflict for the java files due to #8687. You can discard all the conflicts in the HTML resources (use `git checkout --theirs src/test/resources/pages` and stage the HTML files) and re-run godmode after #8745 is merged.